openmc/universe.py

```diff
@@ -714,7 +714,8 @@ def get_unique_universes(self):
                         assert isinstance(u, Universe)
                         univs[u._id] = u
 
-        univs[self._outer._id] = self._outer
+        if self._outer is not None:
```

Comment:
I've been trying to use the property decorators when accessing class attributes. I know this is an inconsistency throughout the API right now, but my preference would be the following:


```python
if self.outer is not None:
    univs[self.outer.id] = self.outer
```